### PR TITLE
GCE: Add functions for Alpha address and forwarding rules

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by freehan** [comment]
just curious, do you need experimental flag for the alpha apis?

**Comment by yujuhong** [comment]
> just curious, do you need experimental flag for the alpha apis?

I was planning to add that later, after your PR gets merged.

File: pkg/cloudprovider/providers/gce/gce_forwardingrule.go [hunk 1]
```diff
@@ -105,6 +113,18 @@ func (gce *GCECloud) CreateRegionForwardingRule(rule *compute.ForwardingRule, re
 	return gce.waitForRegionOp(op, region, mc)
 }
 
+// CreateAlphaRegionForwardingRule creates and returns an Alpha
+// forwarding fule in the given region.
+func (gce *GCECloud) CreateAlphaRegionForwardingRule(rule *computealpha.ForwardingRule, region string) error {
```
Comment (freehan) [inline pkg/cloudprovider/providers/gce/gce_forwardingrule.go]: do you want to do something like this?
```
func (gce *GCECloud) createRegionForwardingRuleInternal(rule gceObject, region string) {
       mc := newForwardingRuleMetricContext("create", region)
       switch v := rule.(type) {
                  .............
       }
       return gce.waitForRegionOp(op, region, mc)
}


CreateRegionForwardingRule(rule *computev1.ForwardingRule, region string) error {
        return createRegionForwardingRuleInternal(rule, region)
}

```

**Comment by freehan** [comment]
/approve

no strong opinion though since you only change one.

File: pkg/cloudprovider/providers/gce/gce_forwardingrule.go (hunk 1 above)
Comment (nicksardo) [inline pkg/cloudprovider/providers/gce/gce_forwardingrule.go]: IMO, that doesn't seem to help.

**Comment by nicksardo** [comment]
@bowei What do you think about specifying "alpha_" or "beta_" in the metrics?

**Comment by bowei** [comment]
Add "alpha_" or  "beta_" to the metrics makes sense.

**Comment by yujuhong** [comment]
> @bowei What do you think about specifying "alpha_" or "beta_" in the metrics?

I actually did that previously in the POC: https://github.com/yujuhong/kubernetes/commit/7d58ea2047d66c97792fed7eb3daaa5a72478392

**Comment by yujuhong** [comment]
> /lgtm I would prefer not renaming compute to computev1 to reduce change

@freehan's previous PRs renamed the imports. I was trying to be consistent. will remove.

**Comment by nicksardo** [comment]
> I actually did that previously in the POC: yujuhong/kubernetes@7d58ea2

I like having its own field even better.

**Comment by yujuhong** [comment]
Reverted the package renaming in the first commit. Added a new commit for recording the version.

**Comment by bowei** [comment]
look ok to me, nick has the final lgtm

**Comment by nicksardo** [comment]
Looks awesome, thanks!
/lgtm

**Comment by yujuhong** [comment]
Forgot about bazel. Updated the BUILD file and re-applied lgtm.

**Pull request merged**